### ISSUE TITLE
Exploratory Data Analysis

### ISSUE THREAD

**Issue by imvs95** [EwoutH/shipping-data#41]
- Additional information on ships (how large are they, etc. etc.)
- Information on ports (number of shipments per day, which goods on which routes typically etc.)

**Comment by EwoutH** [comment]
@thomassandbergen Would you like to do this one?

**Comment by thomassandbergen** [comment]
> @thomassandbergen Would you like to do this one?

Yes, sure. However this will probably quite time intensive in terms of datascraping. Most data is scattered across the internet. I will first finish my datascraper for Maersk

**Comment by imvs95** [comment]
- Boot gegevens: makkelijk via Hapag-Lloyd (voorbeeld: https://www.hapag-lloyd.com/en/online-business/schedule/interactive-schedule/interactive-schedule-solution.html?view=V7030&vessel=46407)
